modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -150,6 +154,12 @@ def show_local_breadcrumb
 
   private
 
+  def ensure_storages_module_active
+    return if OpenProject::FeatureDecisions.storages_module_active?
+
+    raise ActionController::RoutingError, 'Not Found'
+  end
```

Comment:
I hesitated, and thought it would be more interesting to merge it once we use this in another controller, like Bim. It may even take a different approach.